### ISSUE TITLE
the events count in a session detail page is not right

### ISSUE THREAD

**Issue by pinkli** [umami-software/umami#2980]
### Describe the Bug

the count is the same as views, see code https://github.com/umami-software/umami/blob/1b6429f55b2b16ce7b1ff89760b502cef9edbbb8/src/queries/analytics/sessions/getWebsiteSession.ts#L50

should change to `case when website_event.event_type = 1 then 0 else 1 end`

### Database

PostgreSQL

### Which Umami version are you using? (if relevant)

master branch

**Comment by franciscao633** [comment]
Feel free to put in a PR, but it should be the below. We currently only have two event types, but targeting `event_type = 2`, should protect us if we add more. Thanks for catching this.

```
sum(case when website_event.event_type = 2 then 1 else 0 end) as events
```

**Comment by mikecao** [comment]
Fixed in v2.14.0.
